nengo_gui/components/sim_control.py

```diff
@@ -12,6 +12,10 @@
 import nengo_gui.exec_env
 
 class SimControl(Component):
+    """Controls simulation via control node embedded in the neural model.
+    Also instantiates and communitcates with the SimControl and the Toolbar
+    on the JavaScript side, which includes the task of back-end selection."""
```

Comment:
I was just really surprised to find that backend selection was controlled by this class, so I wanted to note it.